lib/git-log.js

```diff
@@ -38,7 +38,9 @@ const tags = require('./tags');
  * See https://git-scm.com/docs/pretty-formats
  */
 exports.GIT_LOG_YAML_PRETTY_FORMAT = [
-  '- subject: >-',
+  '- hash:',
+  '    %H',
```

Comment:
Is there any reason why `%H` is in another line? Now that we're not using the multi line modified, this can be simply `- hash: %H`.